tools/SetupFlow/DevHome.SetupFlow/ViewModels/PackageCatalogListViewModel.cs

```diff
@@ -57,35 +53,30 @@ public partial class PackageCatalogListViewModel : ObservableObject
     /// </summary>
     public async Task LoadCatalogsAsync()
     {
-        if (!_initialized)

```

Comment:
Always attempt  to load the catalogs. The `WinGetPackageCache` service will simply optimize the operation if packages where found in the cache.